client/components/mma/billing/Billing.tsx

```diff
@@ -151,13 +151,10 @@ const BillingRenderer = ([mdaResponse, invoiceResponse]: [
 											productUrlPart:
 												specificProductType.urlPart,
 										}));
-									const resultsPerPage = (
-										paidPlan.billingPeriod ||
-										paidPlan.interval ||
-										''
-									)?.includes('year')
```

Comment:
not sure why this is optional. I've checked the main branch before my refactors and it seems it always has been, but it is required on the type interface.

Reply:
No idea why either, seems fine to leave as it was for now